js/common.js

```diff
@@ -0,0 +1,39 @@
+const EMAIL_PATTERN = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/;
+
+function emailValidChk(email) {
+    if (EMAIL_PATTERN.test(email) === false) {
+        return false;
+    } else {
+        return true;
+    }
+}
```

Comment:
test함수의 반환값이 boolean이다보니 비교로직을 더 가독성있게 변경할 수 있을 것 같아요.

```suggestion
function emailValidChk(email) {
  return EMAIL_PATTERN.test(email)}
```